boxoffice/templates/cash_receipt_new.html.jinja2

```diff
@@ -0,0 +1,557 @@
+{%- extends "layout_email.html.jinja2" -%}
+{%- from "macros_email.html.jinja2" import cta_button -%}
+{%- block content -%}
+  <br/>
+  <hr/>
+  <tr align="right">
+    <td class="receipt-header" colspan="4">
+      <p>{%- trans %}Receipt no. {% endtrans -%}<span>{{ order.receipt_no }}</span></p>
+    </td>
+  </tr>
+  <tr align="left">
+    <td class="invoice-header" colspan="4">
+      <p>{%- trans %}Hasgeek Learning Private Limited {% endtrans -%}</p>
```

Comment:
Get this from the db, as the old template does. We'll deprecate that in a separate PR.